inception-project/inception
# #2111 - Replace okhttp in external recommender with Java HTTP client

**What's in the PR**
- Use Java HTTP client in external recommender
- Switch to mockwebserver3-junit5 dependency
- Switch to a different Matomo/Piwik client library which does not use okhttp
- Remove kotlin Jackson module

**How to test manually**
* Check that there is no warning about a Jackson Kotlin module during startup
* Try setting up a WebLicht recommender and check if it is producing any results
 
**Automatic testing**

**Documentation**
